pyproject.toml

```diff
@@ -55,6 +55,9 @@ docs = [
 cli = [
     "argcomplete"
 ]
+all = [
```

Comment:
Add to README/installation instructions in docs

Reply:
There doesn't seem to be an "installation" section of the built docs, and the README only lists the standard install (no optional deps), and the .[tests] target for running the unittests locally. 

Is it worth adding a more detailed installation section to both in this PR? I can also see the advantage of adding a new .[dislocation] target to add atomman and ovito as optional deps at the same time, but this would require changing the default method for disloc CLE, and building in better error handling for missing imports (i.e. wrapping the import error in an error that says to pip install with the new .[dislocation] optional deps).